rechnung/contract.py

```diff
@@ -1,3 +1,4 @@
+import arrow
```

Comment:
Do we really need arrow and can't stay with the standard datetime lib?

Reply:
Arrow provides lots of functionality (i.e. easy to use ceil() and floor() functions, timedeltas, etc. as well as string-parsing without specifying the format first. I strongly suggest to continue using it, and even replace datetime() completely.